### PR TITLE
Create solution template

### PR DESCRIPTION
**Pull request by iazehaf**

### CODE REVIEW

File: src/ansys/templates/python/solution/hooks/post_gen_project.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+from ansys.templates.utils import keep_files
+
+
+DESIRED_STRUCTURE = []
```
Comment (jorgepiloto) [inline src/ansys/templates/python/solution/hooks/post_gen_project.py:4]: This structure needs to contain the different files you wish your template to include:

```suggestion
DESIRED_STRUCTURE = [
    ".flake8",
    ".github/workflows/ci.yml",
    ...
]
```

It is a bit tedious, but it guarantees that only the files you decide are kept in the final rendered project.

**Review by jorgepiloto — CHANGES_REQUESTED** [review]
This is going in the right direction. This template needs to be added in:

* src/ansys/templates/__init__.py
* src/ansys/templates/cli.py

File: src/ansys/templates/python/solution/hooks/post_gen_project.py (hunk 1 above)
Comment (iazehaf) [reply src/ansys/templates/python/solution/hooks/post_gen_project.py:4]: I see. I though we can pass an empty list which means nothing should be used. I will update the code accordingly.

**Review by jorgepiloto — APPROVED** [review]
This is awesome, @iazehaf. Thanks for adding this new template. Everything looking good to me. Feel free to merge it.

**Review by jorgepiloto — APPROVED** [review]

**Pull request closed without merging**